### PR TITLE
:rocket: Build docs and put into git

### PR DESCRIPTION
**Pull request by bbody**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
     "gh-pages": "gulp build-gh-pages",
     "source-check": "gulp source-check && npm run lint:yaml && npm run lint:travis",
     "setup": "gem install travis -v 1.8.9 --no-rdoc --no-ri && sudo pip install yamllint",
-    "setup_local": "pip install MarkdownPP && npm run setup",
+    "setup_local": "npm run setup && pip install MarkdownPP",
```
Comment (obelic2) [inline package.json]: What happened here?

File: scripts/doc-release.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+PACKAGE_VERSION=$1
+GH_TOKEN=$2
+
+git config --global user.email "support@travis-ci.org"
```
Comment (obelic2) [inline scripts/doc-release.sh]: Shouldn't there be a check if the md files have changed?

File: scripts/doc-release.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+git config --global user.name "Travis CI"
+
+git checkout master
+git reset
+git add CMD-RESUME-DATA-SCHEMA.md README.md CONTRIBUTING.md
```
Comment (obelic2) [inline scripts/doc-release.sh]: Build command? Is it somewhere else?

Comment (bbody) [reply scripts/doc-release.sh]: check Travis config

**Pull request merged**